tests/OperationProcessor.spec.ts

```diff
@@ -40,53 +48,230 @@ async function addBatchOfOneOperation (opBuf: Buffer, cas: Cas, transactionTime:
     batchFileHash: batchFileAddress
   };
 
-  const op = WriteOperation.create(opBuf, resolvedTransaction, 0);
+  const op = WriteOperation.create(opBuf, resolvedTransaction, operationIndex);
   return op;
 }
 
+function getHash (operation: WriteOperation): string {
```

Comment:
Avoid code duplication. Use the same implementation used by Operation Processor. You'll probably need to move it else where to use it. A logical place to put it would be in the WriteOperation class.

Reply:
Good suggestion. Moved it to WriteOperation class